src/game/stats.c

```diff
@@ -104,39 +84,34 @@ static void Stats_CheckTriggers(
 
             const ITEM_INFO *const item = &g_Items[item_num];
 
-            // Add Pierre pickup and kills if oneshot
-            if (item->object_id == O_PIERRE && sector->trigger->one_shot) {
-                Stats_IncludeKillableItem(item_num);
+            if (item->object_id == O_PIERRE) {
+                // Add Pierre pickup and kills if oneshot
```

Comment:
Tiny thing but I think the comment can just say `Add Pierre kill if oneshot`. @lahm86 since the change to de-hardcode item drops, Pierre's pickup in the OG game is tallied from the gameflow? I think it used to be added here which is why the comment references it.

Reply:
Yes, it's handled in `Stats_IncludeKillableItem`.